ambari-agent/src/test/python/ambari_agent/TestCustomServiceOrchestrator.py

```diff
@@ -408,14 +346,15 @@ def side_effect(*args, **kwargs):
       pass
 
   @patch.object(OSCheck, "os_distribution", new = MagicMock(return_value = os_distro_value))
+  @patch.object(ConfigurationBuilder, "get_configuration")
   @patch.object(CustomServiceOrchestrator, "get_py_executor")
   @patch("ambari_commons.shell.kill_process_with_children")
   @patch.object(FileCache, "__init__")
   @patch.object(CustomServiceOrchestrator, "resolve_script_path")
   @patch.object(CustomServiceOrchestrator, "resolve_hook_script_path")
   def test_cancel_backgound_command(self, resolve_hook_script_path_mock,
                                     resolve_script_path_mock, FileCache_mock, kill_process_with_children_mock,
-                                    get_py_executor_mock):
+                                    get_py_executor_mock, get_configuration_mock):
```

Comment:
`test_cancel_background_command` fails for me with:

```
  File "ambari-agent/src/test/python/ambari_agent/TestCustomServiceOrchestrator.py", line 409, in test_cancel_backgound_command
    self.assertTrue(kill_process_with_children_mock.called)
AssertionError: False is not true
```

Reply:
@adoroszlai  can you please provide reproduce of this?

Reply:
fixed

Reply:
Where?

Reply:
I don't remember the exact place. But it is fixed you can re-run it for yourself manually.